biome.jsonc

```diff
@@ -58,20 +50,14 @@
             "deniedGlobals": ["fetch"]
           }
         },
-        // We prefer to use `Math.pow` over `**` operator
-        "useExponentiationOperator": "off",
         // In some cases the enums are initialized with values of other enums
         "useLiteralEnumMembers": "off",
-        // We prefer to have multiple declarations lines
-        "useSingleVarDeclarator": "off",
         // We use `+` operator for string concatenation a lot
         "useTemplate": "off",
         // We use to export types and object without differentiating
         "useExportType": "off",
         // We use to import types and object without differentiating
         "useImportType": "off",
-        // It's nice to use `Number` namespace but should be done in a separate PR
-        "useNumberNamespace": "off",
         // We prefer to auto-initialize enums
         "useEnumInitializers": "off",
         // Namespaces are deprecated way to organize modules in TS
```

Comment:
Upstream?
- noNamespace
- useNamingConvention

Reply:
Moved the `noNamespace` rule to upstream. https://github.com/ChainSafe/lodestar/pull/7763

The `useNamingConvention` is already part of upstream, but here in Lodestar we have a lot of mixup. e.g. We use variables as `"camelCase", "PascalCase", "CONSTANT_CASE"`, which we should not. We must use one consistent pattern for variable names. 

All naming conventions related stuff here is similar mixup. That needs to be fixed over time.